functions.js

```diff
@@ -0,0 +1,197 @@
+/* ## Question 1
```

Comment:
I see that you haven't completed all of the questions. Reach out to me to complete the assignment with my guidance. I'll love to help you out! Im here for you :)